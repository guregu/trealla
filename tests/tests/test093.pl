:-initialization(main).

main :-
	write_term([[_|_]|_],[max_depth(1)]), nl,
	write_term(+A - +B,[max_depth(2)]), nl,
	write_term(+A - +B - +C,[max_depth(2)]), nl,
	write_term(-_* -_,[max_depth(2)]), nl,
	write_term([]*[]*[],[max_depth(3)]), nl,
	write_term_to_chars([1|(A*[[]*B])],[quoted(true),max_depth(5)],K),
	true.
