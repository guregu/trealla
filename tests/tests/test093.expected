[[...|[...]]|_5]
+_6- +_7
+_6- +_7- +_8
(-_9)* -_10
[]*[]*[]
